src/main/java/bio/terra/service/filedata/google/gcs/GcsPdao.java

```diff
@@ -102,11 +101,16 @@ public Storage storageForBucket(GoogleBucketResource bucketResource) {
         .flatMap(blob -> getBlobLinesStream(blob, projectId, storage));
   }
 
-  @SuppressFBWarnings("OS_OPEN_STREAM")
   private static Stream<String> getBlobLinesStream(Blob blob, String projectId, Storage storage) {
     logger.info(String.format("Reading lines from %s", getGsPathFromBlob(blob)));
     var reader = storage.reader(blob.getBlobId(), Storage.BlobSourceOption.userProject(projectId));
-    return new BufferedReader(Channels.newReader(reader, StandardCharsets.UTF_8)).lines();
+    try (var bReader = new BufferedReader(Channels.newReader(reader, StandardCharsets.UTF_8))) {
+      return bReader.lines();
```

Comment:
Does this work? I would expect the `try` to close the `BufferedReader` and on the first call to the lines stream you'd hit the error check in `BufferedReader.ensureOpen()`.

Reply:
It does not work.